contrib/cfgchanged.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Hook for oxidized to initate validation after a changed config is stored
+# 
+# $OX_EVENT
+# $OX_NODE_NAME
+# $OX_NODE_IP
+# $OX_NODE_FROM
+# $OX_NODE_MSG
+# $OX_NODE_GROUP
+# $OX_NODE_MODEL
+# $OX_JOB_STATUS
+# $OX_JOB_TIME
+# $OX_REPO_COMMITREF
+# $OX_REPO_NAME
+
+Storage="data"
+MyPath=`dirname $0`
+FilePath="$MyPath/$Storage/$OX_NODE_GROUP"
+GitCmd=`which git`
+
+# Clone the last version of the config
+if [ -d "$FilePath" ]; then
+  # Pull
+  cd "$FilePath" && $GitCmd pull
+else
+  # Clone
+  mkdir -p $FilePath
+  $GitCmd clone "$OX_REPO_NAME" "$FilePath"
+fi
+
+# No run the validator
+$MyPath/validate_config.sh  "$MyPath/$Storage/$OX_NODE_GROUP/$OX_NODE_NAME" "$OX_NODE_MODEL"
```

Comment:
The goal I have in mind with this is to have the oxidized integration be 100% ovious just from the contrib files and the documentation. Currently I can't really tell what `validate_config.sh` might contain, or did I miss anything? I assume it calls netlint and then does something with the result? We would wan't to include that file as well.

Reply:
Think you are right here. Will set-up a template for that file too

Reply:
done